erpnext/healthcare/doctype/patient/patient.json

```diff
@@ -351,13 +350,47 @@
    "fieldname": "middle_name",
    "fieldtype": "Data",
    "label": "Middle Name (optional)"
+  },
+  {
+   "collapsible": 1,
+   "fieldname": "customer_details_section",
+   "fieldtype": "Section Break",
+   "label": "Customer Details"
+  },
+  {
+   "fieldname": "customer_group",
+   "fieldtype": "Link",
+   "label": "Customer Group",
+   "options": "Customer Group"
+  },
+  {
+   "fieldname": "territory",
+   "fieldtype": "Link",
+   "label": "Territory",
+   "options": "Territory"
+  },
+  {
+   "fieldname": "column_break_24",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "default_price_list",
+   "fieldtype": "Link",
+   "label": "Default Price List",
+   "options": "Price List"
+  },
+  {
+   "fieldname": "language",
+   "fieldtype": "Link",
+   "label": "Print Language",
+   "options": "Language"
```

Comment:
Awesome fields added to the Patient DocType! But I think we can hide all other fields in the Customer Details section if the user selects an existing customer.  They also should be `set_only_once` since the customer is getting created on the `after_insert` event and we are not updating the customer on saving the patient document.

![customer-details](https://user-images.githubusercontent.com/24353136/80035242-5710a880-850d-11ea-9941-ab8335481dca.png)